lib/article-tile/test/index.test.js

```diff
@@ -43,10 +43,18 @@ describe('Component', function () {
   };
 
   describe('<ArticleTile />', function () {
+    const wrapper = shallow(<ArticleTile {...props}/>);
+    const children = wrapper.children().nodes;
     it('should render our ArticleTile component', function (done) {
-      const wrapperTile = shallow(<ArticleTile {...props}/>);
-      const childrenTile = wrapperTile.children().nodes;
-      expect(childrenTile).to.have.length(3);
+      expect(children).to.have.length(3);
+      done();
+    });
+    it('should render the correct child components', function (done) {
+      expect(wrapper.find('div')).to.have.length(5);
+      expect(wrapper.find('FadeImage')).to.have.length(1);
+      expect(wrapper.find('h2')).to.have.length(1);
+      expect(wrapper.find('h3')).to.have.length(1);
+      expect(wrapper.find('h4')).to.have.length(1);
```

Comment:
These tests feel a bit arbitrary. 

* Containing exactly 5 `<div/>`s isn't really something we care about in the slightest, and seems like it would be very fragile in future code changes, so would just add to maintenance cost without really adding any value in terms of confirming correct behaviour.
* If we're going to assert on the headers being present, then I'd probably want to also assert that they contain the text we expect.